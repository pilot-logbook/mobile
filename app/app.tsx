import React, { Component } from 'react';
import { Text, View } from 'react-native';

// tslint:disable-next-line:no-shadowed-variable
export const App = class App extends Component {
  public render() {
    return (
      <View style={{ flex: 1, justifyContent: 'center', backgroundColor: 'white' }}>
        <Text>Hello Pilog!</Text>
      </View>
    );
  }
}
