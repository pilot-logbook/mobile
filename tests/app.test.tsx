import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';
import { App } from '../app/app';

describe('app', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
})
