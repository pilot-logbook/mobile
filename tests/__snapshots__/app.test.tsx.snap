// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Hello Pilog!
  </Text>
</View>
`;
